geometry_msgs/Point query
---
geometry_msgs/PoseArray waypoints
